eledina/api/api_blueprint.py

```diff
@@ -244,3 +245,29 @@ def login():
     return jsonify_response(payload)
 
 
+@api.route("/blog/new", methods=["POST"])
+@ip_rate_limit
+def blog_new():
+    print(request.data)
+    body = loads(request.data)
+
+    title = body.get("title")
+    content = body.get("content")
+    date = body.get("date")
+
+    # Class and function imported from models.py
+    blogs.upload_blog(title, content, date)
+    blogpack = {
+        "status": JsonStatus.OK,
+    }
+    return jsonify_response(blogpack)
+    # CacheGenerator().blog_cache()
+
+
+@api.route("/blog/list", methods=["GET"])
+@ip_rate_limit
+def blog_get():
```

Comment:
Zaradi berljivosti na front-endu (url_for in te fore) predlagam, da preimenuješ to funkcijo.